### PR TITLE
Fixes #1072: Published view header is not centered

### LINKED ISSUE #1072: Published view header is not centered
You can test this locally at `localhost:8080/publish-assets`

image

### PR DESCRIPTION
**Pull request by gesa**
Since there's no real way to test this locally, I'm only about 95% certain this is the right fix to #1072.

### CODE REVIEW

**Comment by gesa** [comment]
(see also #1159)

**Comment by k88hudson** [comment]
Hrm, did you forget to add the css file or something? :stuck_out_tongue_winking_eye:

**Comment by gesa** [comment]
No, the thing that was throwing the title off-center was the inclusion of all the divs in the navigation bar.

**Comment by k88hudson** [comment]
@gesa this is just a plain `h1` though, like no colours/header/anything

**Comment by gesa** [comment]
@k88hudson unfortunately, I have no way of testing this, due to the fact that this code isn't executed anywhere locally. (at least, not anywhere that @thisandagain and I could figure out)

**Comment by k88hudson** [comment]
@gesa oops, sorry – should have put this in the ticket! you can test at  http://localhost:8080/publish-assets

**Comment by gesa** [comment]
@k88hudson Yeah, that was just giving me a white screen when I tried.

File: components/publishHeader/index.less [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#publish-header {
+    background-color: @turquoise;
```
Comment (k88hudson) [inline components/publishHeader/index.less]: i think this name is kinda deprecated, use `@highlight` instead

**Comment by k88hudson** [comment]
Nice! looking smoooth :surfer:

**Pull request merged**